packages/terra-application-navigation/src/terra-dev-site/test/application-navigation/HeroDrawer.test.jsx

```diff
@@ -0,0 +1,17 @@
+
+/* eslint-disable */
+import React from 'react';
+import Application from 'terra-application';
+import ApplicationNavigation from '../../../ApplicationNavigation';
+import Placeholder from 'terra-doc-template/lib/Placeholder';
+
+const hero = <Placeholder title="Test Hero" variant="light" />;
+
+// TODO: remove terra-application after it is incorporated into dev-site
+const TitleTest = () => (
+  <Application locale="en">
+    <ApplicationNavigation hero={hero} />
```

Comment:
This test doesn't seem to do anything different than `Hero.test.jsx` other than make the placeholder all white